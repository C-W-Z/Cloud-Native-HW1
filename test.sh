#!/bin/sh

python3 ./test/clean.py
python3 ./cli.py < ./test/test.in > ./test/test.out
python3 ./test/clean.py
pytest -s test/pytest_cloudshop.py
