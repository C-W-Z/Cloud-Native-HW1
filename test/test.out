Success
100001
Phone model 8|Black color, brand new|1000|2025-03-23 21:22:08|Electronics|user1
100002
Success
Error - user already existing
100003
T-shirt|White color|20|2025-03-23 21:22:08|Sports|user2
Error - category not found
Black shoes|Training shoes|100|2025-03-23 21:22:08|Sports|user1
T-shirt|White color|20|2025-03-23 21:22:08|Sports|user2
Sports
Error - listing owner mismatch
Success
Electronics
Sports
Success
Electronics
Error - unknown user
